modules/psap-special-resource-operator-using-CRD.adoc

```diff
@@ -0,0 +1,480 @@
+                                spec:
+                                    serviceAccount: {{.Values.specialResourceModule.metadata.name}}
+                                    serviceAccountName: {{.Values.specialResourceModule.metadata.name}}
+                                    containers:
+                                    - image: {{.Values.registry}}/{{.Values.specialResourceModule.metadata.name}}-{{.Values.groupName.driverContainer}}:{{.Values.kernelFullVersion}}
+                                      name: {{.Values.specialResourceModule.metadata.name}}
+                                      imagePullPolicy: Always
+                                      command: ["/sbin/init"]
+                                      lifecycle:
+                                        preStop:
+                                            exec:
+                                                command: ["/bin/sh", "-c", "systemctl stop kmods-via-containers@{{.Values.specialResourceModule.metadata.name}}"]
+                                            securityContext:
+                                                privileged: true
+----
+.. Save this YAML template for the placement of policies in the `templates` directory as `0003-policy.yaml`.
++
+[source,yaml]
+----
+apiVersion: apps.open-cluster-management.io/v1
+kind: PlacementRule
+metadata:
+    name: {{.Values.specialResourceModule.metadata.name}}-placement
+spec:
+    clusterConditions:
+    - status: "True"
+      type: ManagedClusterConditionAvailable
+    clusterSelector:
+      matchExpressions:
+      - key: name
+        operator: NotIn
+        values:
+        - local-cluster
+---
+apiVersion: policy.open-cluster-management.io/v1
+kind: PlacementBinding
+metadata:
+    name: {{.Values.specialResourceModule.metadata.name}}-binding
+placementRef:
+    apiGroup: apps.open-cluster-management.io
+    kind: PlacementRule
+    name: {{.Values.specialResourceModule.metadata.name}}-placement
+subjects:
+    - apiGroup: policy.open-cluster-management.io
+      kind: Policy
+      name: policy-{{.Values.specialResourceModule.metadata.name}}-ds
+----
++
+.. Change into the `charts/acm-simple-kmod-0.0.1` directory:
++
+[source,terminal]
+----
+cd ..
+----
+.. Save the following YAML for the chart as `Chart.yaml` in the `charts/acm-simple-kmod-0.0.1` directory:
++
+[source,yaml]
+----
+apiVersion: v2
+name: acm-simple-kmod
+description: Build ACM enabled simple-kmod driver with SpecialResourceOperator
+icon: https://avatars.githubusercontent.com/u/55542927
+type: application
+version: 0.0.1
+appVersion: 1.6.4
+----
+
+. Create a config map to store the chart files:
+.. Create a directory for the config map files:
++
+[source,terminal]
+----
+$ mkdir cm
+----
+.. Copy the Helm chart into the `cm` directory:
++
+[source, terminal]
+----
+$ cp acm-simple-kmod-0.0.1.tgz cm/acm-simple-kmod-0.0.1.tgz
```

Comment:
I am missing the step to create the tgz, or did I skip it?

Reply:
The command is `helm package <directory> -d <destination>`. In this case it would be `helm package charts/acm-simple-kmod-0.0.1 -d cm/`

Reply:
Done